tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -484,6 +661,44 @@ class MklConcatOp : public OpKernel {
         output_tensor->flat<uint8>().size() * sizeof(uint8));
   }
 
+  void CallQuantizedEigenVersion(OpKernelContext* context,
+                                 const OpInputList& values,
+                                 const OpInputList& input_mins,
+                                 const OpInputList& input_maxes,
+                                 const MklDnnShapeList& mkl_input_shapes) {
+    CHECK_EQ(values.size(), mkl_input_shapes.size());
+
+    std::vector<Tensor> converted_values;
```

Comment:
Since in both cases (```if``` and ```else```) converted_values gets elements pushed in. You should reserve it before entering the ```for``` loop. Perhaps to size ```mkl_input_shapes.size()```.

Reply:
Done. Thanks!